src/goto-programs/goto_program_irep.cpp

```diff
@@ -78,7 +91,7 @@ void convert(const goto_programt &program, irept &irep)
   irep.hide(program.hide);
 }
 
-void convert(const irept &irep, goto_programt &program)
+void convert(const irept &irep, goto_programt &program, const contextt &context)
```

Comment:
why do we need to pass the context here? It should be already filled when we reach the process_goto call.

Reply:
The context has indeed been filled when we reach process_goto call. The reason of passing it is to intitialize the namespacet pointer `migrate_namespace_lookup` in `migrate.cpp`.